# Parameters for all agents

agent_1:
  start_pos:
    x: 0.0
    y: 0.0
  goal_pos:
    x: 15.0
    y: 15.0
  goal_dist: 1.0
  rrt_iters: 100
  step_size: 5.0
  near_radius: 30.0
  spin_rate: 50

agent_2:
  start_pos:
    x: 0.0
    y: 1.0
  goal_pos:
    x: 0.0
    y: 10.0
  goal_dist: 2.0
  rrt_iters: 10
  step_size: 1.0
  spin_rate: 10

agent_3:
  start_pos:
    x: 0.0
    y: 1.0
  goal_pos:
    x: 0.0
    y: 5.0
  goal_dist: 2.0
  rrt_iters: 10
  step_size: 1.0
  spin_rate: 10

agent_4:
  start_pos:
    x: 0.0
    y: 1.0
  goal_pos:
    x: 0.0
    y: 5.0
  goal_dist: 2.0
  rrt_iters: 10
  step_size: 1.0
  spin_rate: 10

agent_5:
  start_pos:
    x: 0.0
    y: 1.0
  goal_pos:
    x: 0.0
    y: 5.0
  goal_dist: 2.0
  rrt_iters: 10
  step_size: 1.0
  spin_rate: 10
